src/virtualenv/util/path/_pathlib/via_os_path.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
Maybe it's worth calling out the reason this exists in a comment? I assume it's https://github.com/mcmtroffaes/pathlib2/issues/56

Reply:
It's already there where it's used https://github.com/pypa/virtualenv/blob/rewrite/src/virtualenv/util/path/_pathlib/__init__.py#L33